generator/common/beatgen/beatgen.go

```diff
@@ -101,6 +101,12 @@ func Generate() error {
 		return err
 	}
 
+	// Make sure the ElasticBeatsDir value is cached
```

Comment:
A little more fallout: previously because findElasticBeatsDir was being called at package-init, it was always cached. With my other fix, that's no longer the case, and we need to cache it here as otherwise the setup.InitModule target fails.